src/scss/components/_inputs.scss

```diff
@@ -70,6 +70,10 @@ $input-select-margin-right: 1;
 .usa-checkbox__input:disabled + .usa-checkbox__label {
   color: color('disabled');
   border-color: color('disabled');
+
+  .usa-checkbox__image {
+    filter: invert(45%) sepia(1%) saturate(0%) hue-rotate(232deg) brightness(100%) contrast(80%);
+  }
```

Comment:
Are these styles necessary, since it appears they're the same as what would be applied below on lines 144-150?

Reply:
Not necessary, removing now. Thanks for catching.